src/kOS.Safe/Encapsulation/EnumerableValue.cs

```diff
@@ -1,27 +1,27 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
-using System.Linq;
 using kOS.Safe.Encapsulation.Suffixes;
 using kOS.Safe.Serialization;
+using System.Linq;
 
 namespace kOS.Safe.Encapsulation
 {
-    public abstract class EnumerableValue<T, TC> : Structure, IEnumerable<T>, IDumper where TC : IEnumerable<T>
+    public abstract class EnumerableValue<T, TE> : Structure, IEnumerable<T>, IDumper where TE : IEnumerable<T>
     {
-        protected TC Collection { get; private set; }
+        protected TE Enum { get; private set; }
```

Comment:
Ah, I was just noticing that too.  I guess I'll have to jump ahead on the PR list then.